.isort.cfg

```diff
@@ -0,0 +1,3 @@
+[settings]
+multi_line_output=3
+include_trailing_comma=true
```

Comment:
I'd also like line_length = 80 because that enforces multiline imports sooner, but decided not to impose too many of my own opinions

Reply:
That's the default, isn't it?

Reply:
hmm. default is 79. For some reason I thought that it was picking up HQ's stickler 115 line length. I could just be crazy